sql/core/src/test/java/test/org/apache/spark/sql/JavaDataFrameSuite.java

```diff
@@ -523,4 +523,55 @@ public void testUDF() {
       .map(row -> row.get(0).toString() + row.getString(1)).toArray(String[]::new);
     Assert.assertArrayEquals(expected, result);
   }
+
+  /**
+   * Tests the Java API of Observation and Dataset.observe(Observation, Column, Column*).
+   */
+  @Test
+  public void testObservation() {
+    Observation namedObservation = new Observation("named");
+    Observation unnamedObservation = new Observation();
+
+    Dataset<Long> df = spark
+            .range(100)
```

Comment:
Actually we should keep it with 2 space indentation here same as Scala side.

Reply:
fixed